global-files/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,82 @@
+
+## The simple version
+
+- Be empathetic
+- Be welcoming
+- Be kind
+- Be honest
+- Be supportive
+- Be polite
+
+## The details
+
+Exercism should be a safe place for everybody regardless of
+
+- Gender, gender identity or gender expression
+- Sexual orientation
+- Disability
+- Physical appearance (including but not limited to body size)
+- Race
+- Age
+- Religion
+- Anything else you can think of.
+
+As someone who is part of this community, you agree that:
+
+- We are collectively and individually committed to safety and inclusivity.
+- We have zero tolerance for abuse, harassment, or discrimination.
+- We respect people’s boundaries and identities.
+- We refrain from using language that can be considered offensive or oppressive (systemically or otherwise), eg. sexist, racist, homophobic, transphobic, ableist, classist, etc. - this includes (but is not limited to) various slurs.
+- We avoid using offensive topics as a form of humour.
+
+We actively work towards:
+
+- Being a safe community
+- Cultivating a network of support & encouragement for each other
+- Encouraging responsible and varied forms of expression
+
+We condemn:
+
+- Stalking, doxxing, or publishing private information
+- Violence, threats of violence or violent language
+- Anything that compromises people’s safety
+- Conduct or speech which might be considered sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory or offensive in nature.
+- The use of unwelcome, suggestive, derogatory or inappropriate nicknames or terms.
+- Disrespect towards others (jokes, innuendo, dismissive attitudes) and towards differences of opinion.
+- Intimidation or harassment (online or in-person). Please read the [Citizen Code of Conduct](https://github.com/stumpsyn/policies/blob/master/citizen_code_of_conduct.md) for how we interpret harassment.
+- Inappropriate attention or contact.
+- Not understanding the differences between constructive criticism and disparagement.
+
+These things are NOT OK.
+
+Be aware of how your actions affect others. If it makes someone uncomfortable, stop.
+
+If you say something that is found offensive, and you are called out on it, try to:
+
+- Listen without interruption.
+- Believe what the person is saying & do not attempt to disqualify what they have to say.
+- Ask for tips / help with avoiding making the offence in the future.
+- Apologise and ask forgiveness.
+
+## History
+
+This policy was initially adopted from the Front-end London Slack community and has been modified since. A version history can be seen on [GitHub](https://github.com/exercism/website-copy/edit/main/pages/code_of_conduct.md).
+
+_This policy is a "living" document, and subject to refinement and expansion in the future. This policy applies to the Exercism website, the Exercism GitHub organisation, any other Exercism-related communication channels (e.g. Slack, Twitter email) and any other Exercism entity or event._
```

Comment:
```suggestion
_This policy is a "living" document, and subject to refinement and expansion in the future. This policy applies to the Exercism website, the Exercism GitHub organisation, any other Exercism-related communication channels (e.g. Slack, Twitter, Email) and any other Exercism entity or event._
```

Reply:
Why use a capital E for email?

```suggestion
_This policy is a "living" document, and subject to refinement and expansion in the future. This policy applies to the Exercism website, the Exercism GitHub organisation, any other Exercism-related communication channels (e.g. Slack, Twitter, email) and any other Exercism entity or event._
```

Reply:
comma is good, but it should be 'email' (as it's a "common noun" in english now)